packages/eslint-plugin-template/tests/rules/attributes-order/cases.ts

```diff
@@ -338,4 +339,64 @@ export const invalid = [
                    
     `,
   }),
+  convertAnnotatedSourceToFailureCase({
+    messageId,
+    description: 'should work with structural directive and single let',
+    annotatedSource: `
+      <td mat-cell *matCellDef="let element"></td>
+          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    `,
+    options: [{ alphabetical: true }],
+    data: {
+      expected: '`*matCellDef`, `mat-cell`',
+      actual: '`mat-cell`, `*matCellDef`',
+    },
+    annotatedOutput: `
+      <td *matCellDef="let element" mat-cell></td>
+          
+    `,
+  }),
+  convertAnnotatedSourceToFailureCase({
+    messageId,
+    description:
+      'should work with structural directive and single let spread over lines',
+    annotatedSource: `
+      <td mat-cell *matCellDef="
+          ~~~~~~~~~~~~~~~~~~~~~~
+        let element
+        ~~~~~~~~~~~
+      "></td>
+      ~
+    `,
+    options: [{ alphabetical: true }],
+    data: {
+      expected: '`*matCellDef`, `mat-cell`',
+      actual: '`mat-cell`, `*matCellDef`',
+    },
+    annotatedOutput: `
+      <td *matCellDef="
+          
+        let element
+        
+      " mat-cell></td>
+      
+    `,
+  }),
+  convertAnnotatedSourceToFailureCase({
+    messageId,
+    description: 'should work with structural directive with as',
```

Comment:
Suggesting this change because structural directives with `as` have been working before this PR. Only when a trailing whitespace exists does the auto-fixer break.
```suggestion
    description: 'should work with structural directive with as and trailing whitespace',
```